### DISCUSSION TITLE
Disable online indication in message listview in Android

### DISCUSSION THREAD

**Discussion by ramanareddy463** [GetStream/stream-chat-android#4661] [General]
How to disable/remove online indication on AvatherView in message listview (XML) in android kotlin.

**Comment by filbabic** [comment]
Hey there @ramanareddy463 !

You can use our offline support configuration to disable `userPresence`. This should disable any indication if you don't need it in your app.

https://getstream.io/chat/docs/sdk/android/client/guides/offline-support/#configuration

**Reply by ramanareddy463** [reply]
Thanks for reply @filbabic.
I don't want to show green color circle on avather view.
`userPresence` should be enable.

**Reply by filbabic** [reply]
@ramanareddy463 `userPresence` is only used to show if the user is online or not, in the indicator.

Do you need the online status of people, but don't want to show the green circle, or do you not need this feature at all?

**Reply by ramanareddy463** [reply]
@filbabic yes, I want online status of people, but don't want to show the green circle.

**Reply by filbabic** [reply]
Hey @ramanareddy463 !

We have a boolean property in our styles which you can customize.

If you add this `streamUiAvatarOnlineIndicatorEnabled` to your styles/attributes  and set that to `false`, the indicators will be removed in the SDK UI. But the presence will remain.

For example, we have this in our SDK sample app:

```xml
    <style name="StreamUiMessageListAvatarStyle" parent="StreamUiBaseAvatarStyle">
        <item name="android:layout_width">@dimen/stream_ui_avatar_size_small</item>
        <item name="android:layout_height">@dimen/stream_ui_avatar_size_small</item>
        <item name="streamUiAvatarTextSize">@dimen/stream_ui_avatar_text_size_small</item>
        <item name="streamUiAvatarOnlineIndicatorEnabled">false</item> // here
    </style>
```

Notice how we just override it and set it to false. This is an example within a specific style, but you should be able to create this attribute in your attrs or values XML file:

```xml
<item name="streamUiAvatarOnlineIndicatorEnabled" type="bool">false</item>
```

**Reply by ramanareddy463** [reply]
Thanks for the quick reply. @filbabic 
able to disable online indication(green color circle).

**Reply by filbabic** [reply]
Great to hear that, thank you so much!
